Assets/StreamingAssets/Quests/20C00Y00.txt

```diff
@@ -169,8 +170,11 @@ _1stparton_ task:
 	end quest 
 
 variable _2ndparton_
+--unused variable
 _clearclick_ task:
-	when _qgclicked_ and not _2ndparton_ 
+	when _qgclicked_ and not _mondead_ 
+	--erroneous variable use
```

Comment:
I find this sort of commenting a bit unclear. Once the line has been fixed, the comment is no longer relevant. Once the file is merged, it will just seem to indicate a problem that's no longer there and will become confusing later.

Reply:
I'm only using comments so far to justify the changes I'm making. Once everyone's satisfied I don't see a need for them, and I figured it'd be just as easy to remove them in the future. Once a quest is fixed and working, I don't see a need for artifact comments from the past to remain in them.